apps/angular-thirty-seconds/src/app/angular-thirty-seconds/create-snippet/snippet-modal/snippet-overview.component.ts

```diff
@@ -17,13 +17,9 @@ function getTagsStringList(tagsArray: Array<string>): string {
 function getSnippet(value): string {
   return `
 ---
-
 title: ${value.title}
-
```

Comment:
You'd better to leave this spaces. Or it looks like this: 
![image](https://user-images.githubusercontent.com/40883170/54480262-4801f600-4837-11e9-86fc-5f935282178a.png)